### ISSUE TITLE
Does not support sync function signature

### ISSUE THREAD

**Issue by felixfbecker** [enriched/typed-jsonwebtoken#9]

**Comment by enriched** [comment]
Thanks for getting on that, and it appears that the typings registry is updated to point at your current version, so this is no longer a problem?

**Comment by felixfbecker** [comment]
Yeah apparently I build new typings because there was a lot of stuff not up-to-date. Your typings are still referenced for v5: https://github.com/typings/registry/blob/master/npm/jsonwebtoken.json

I can give you push access to the repo if you want to contribute on it instead.
